### ISSUE TITLE
Add Signal to the list of instant messaging providers

### ISSUE THREAD

**Issue by ostasevych** [nextcloud/contacts#2910]
### Is your feature request related to a problem? Please describe.

By now it is not possible to indicate Signal profile which a contact uses.

### Describe the solution you'd like

Please, extend the list of instant messaging providers by adding Signal Private Messenger and the handler. Therefore a user may click over it and in case the signal app is installed it will be launched. 
![image](https://user-images.githubusercontent.com/5199030/188259149-c3d976e2-89d4-469b-bc18-45336c795140.png)

### Describe alternatives you've considered

The main big alternative is to let a user to indicate own instant messaging providers in the config.php file.

**Comment by call-me-matt** [comment]
Hi. I believe Signal uses the phone number. Or is there a dedicated account for signal?

**Comment by ostasevych** [comment]
> Hi. I believe Signal uses the phone number. Or is there a dedicated account for signal?

Sure, it uses it. However, a user may use a special phone number specifically for Signal, and secondly, I am about the handler which launches the signal app installed on a PC, if any.

**Comment by ostasevych** [comment]
The handler may allow a user to select the app and the action: call or chat, in most cases.

**Comment by rayfun** [comment]
> Hi. I believe Signal uses the phone number. Or is there a dedicated account for signal?

Signal also now provides discovery by id.
Please add Signal to the list of instant messengers.

https://signal.org/blog/phone-number-privacy-usernames/

https://github.com/nextcloud/contacts/blob/5102763ec90d0c2e2ee0662d03de28952909740a/src/models/rfcProps.js#L211

**Comment by call-me-matt** [comment]
fair point. I'll create a pull request to add Signal to the list. Looking at the title of this issue, it should be solved by that. Feel free to open it again (or the duplicate issue referenced above) in case you want to go further with the integration.
